Makefile

```diff
@@ -20,34 +20,41 @@ format:
 	@go run $(gosimports) -w $(shell find . -name '*.go' -type f)
 
 .PHONY: test
-test:
+test: copy-wasm-examples
 	@(cd wasmprocessor; go test -v -tags docker ./...)
 	@(cd wasmexporter; go test -v -tags docker ./...)
 	@(cd wasmreceiver; go test -v -tags docker ./...)
 	@(cd guest; go test -v -tags docker ./...)
 
+.PHONY: examples/processor/nop/main.wasm
```

Comment:
Difficult to track all changes in Go with Makefile.
Go itself has the cache mechanism, so it's OK to build it everytime.